builtin/game/auth.lua

```diff
@@ -180,6 +188,20 @@ core.set_player_privs    = auth_pass("set_privileges")
 core.remove_player_auth  = auth_pass("delete_auth")
 core.auth_reload         = auth_pass("reload")
 
+function core.change_player_privs(name, changes)
+	local privs = core.get_player_privs(name)
+	for priv, change in pairs(changes) do
+		if change == true then
+			privs[priv] = true
+		elseif change == false then
+			privs[priv] = nil
+		else
+			error("non-bool value in `changes` table")
```

Comment:
```suggestion
			error("non-bool value given to `minetest.change_player_privs`")
```

same as above

Reply:
I don't think this is really needed given that the stack trace should properly mention `change_player_privs`.

(There are contrived scenarios where this would make a minor difference to the effect of "Lua doesn't know what to call this function" but we don't really have to care about them)

Reply:
I see the error message first, before I see the stacktrace, so it should be as useful as possible. If the error message already contains the offending function call, I can often fix the error without even looking at the stacktrace. That saves time.